src/mopidy/backend.py

```diff
@@ -137,13 +137,26 @@ def get_images(self, uris: list[Uri]) -> dict[Uri, list[Image]]:
         """
         return {}
 
+    def lookup_many(self, uris: list[Uri]) -> dict[Uri, list[Track]]:
+        """See :meth:`mopidy.core.LibraryController.lookup`.
+
+        *MUST be implemented by subclass.*
+        """
+
+        results = {}
+        for uri in uris:
+            results[uri] = self.lookup(uri)
+
+        return results
+
     def lookup(self, uri: Uri) -> list[Track]:
         """See :meth:`mopidy.core.LibraryController.lookup`.
 
         *MUST be implemented by subclass.*
```

Comment:
What's the convention on deprecating this method? Mark it as deprecated here in the comment for apidoc with a pointer to implement `lookup_many`?

Reply:
Yes, look for calls to `deprecation.warn("blah")` elsewhere.

Reply:
We can sort this out afterwards, if you like? i.e. do we mark it pending or really deprecated?

Reply:
Sure thing, I'm happy to get this shipped now :)

Reply:
For now, let's just update the docstring a bit:

```suggestion
        *MUST be implemented by subclass if :meth:`lookup_many` is not implemented.*
```